### PR TITLE
Fixed Rendering Order of Background Color and Background Image

### PR DESCRIPTION
**Pull request by thequixotic**
Let me know for any changes.
Please do thorough testing :)

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/Form.java [hunk 1]
```diff
@@ -681,16 +681,9 @@ public int BackgroundColor() {
       defaultValue = Component.DEFAULT_VALUE_COLOR_WHITE)
   @SimpleProperty
   public void BackgroundColor(int argb) {
-    backgroundColor = argb;
-    if (argb != Component.COLOR_DEFAULT) {
-      viewLayout.getLayoutManager().setBackgroundColor(argb);
```
Comment (thequixotic) [inline appinventor/components/src/com/google/appinventor/components/runtime/Form.java]: setBackgroundColor(argb) is same as setBackgroundDrawable(new ColorDrawable(argb))
Android does not seem to provide a way to setbackgroundcolor and setbackgroundimage simultaneously.
It treats a color as an Image (Drawable) . 
It does not differentiate between a color and an image both are drawables to android.
Which is why setting color would override the background image.
Its like assigning two values to the same variable, the last one assigned would prevail.
x = image;
x = color; 
color wins here. unless we blend both values when setting color...

**Comment by jisqyv** [comment]
LGTM

**Comment by jisqyv** [comment]
@thequixotic I slightly reformated the setBackground function just a little bit.

**Comment by jisqyv** [comment]
@thequixotic MERGED into the ucr branch for the next Component Release.

**Pull request closed without merging**